docs/changelog.txt

```diff
@@ -1,6 +1,82 @@
 =========
 
+v1.5 (03/27/2015)
+-----------------
+
+`Full Changelog <https://github.com/django-compressor/django-compressor/compare/1.4...HEAD>`_
+
+- Fix typo and restyle manage command examples `\#604 <https://github.com/django-compressor/django-compressor/pull/604>`_ (`alimony <https://github.com/alimony>`_)
+
+- Fix compress command for 1.8 `\#601 <https://github.com/django-compressor/django-compressor/pull/601>`_ (`carltongibson <https://github.com/carltongibson>`_)
+
+- Fix build `\#599 <https://github.com/django-compressor/django-compressor/pull/599>`_ (`carltongibson <https://github.com/carltongibson>`_)
+
+- Update settings.txt `\#584 <https://github.com/django-compressor/django-compressor/pull/584>`_ (`johnfraney <https://github.com/johnfraney>`_)
+
+- Fix Django 1.8 warnings  `\#583 <https://github.com/django-compressor/django-compressor/pull/583>`_ (`sdfsdhgjkbmnmxc <https://github.com/sdfsdhgjkbmnmxc>`_)
+
+- Handle TypeError from import\_module `\#571 <https://github.com/django-compressor/django-compressor/pull/571>`_ (`blueyed <https://github.com/blueyed>`_)
+
+- Fix reading UTF-8 files which have BOM `\#568 <https://github.com/django-compressor/django-compressor/pull/568>`_ (`JereMalinen <https://github.com/JereMalinen>`_)
+
+- shlex.quote/pipes.quote is not compatible with Windows `\#561 <https://github.com/django-compressor/django-compressor/pull/561>`_ (`thenewguy <https://github.com/thenewguy>`_)
+
+- Run automated tests on Django 1.7 `\#556 <https://github.com/django-compressor/django-compressor/pull/556>`_ (`frewsxcv <https://github.com/frewsxcv>`_)
+
+- Patch 1 `\#555 <https://github.com/django-compressor/django-compressor/pull/555>`_ (`francescozaia <https://github.com/francescozaia>`_)
+
+- Fix AttributeError in offline compression `\#554 <https://github.com/django-compressor/django-compressor/pull/554>`_ (`cbjadwani <https://github.com/cbjadwani>`_)
+
+- Support for clean-css `\#551 <https://github.com/django-compressor/django-compressor/pull/551>`_ (`vovanbo <https://github.com/vovanbo>`_)
+
+- Types at readme \(-engine should be --engine\) `\#540 <https://github.com/django-compressor/django-compressor/pull/540>`_ (`lukaszb <https://github.com/lukaszb>`_)
+
+- Fix typo `\#539 <https://github.com/django-compressor/django-compressor/pull/539>`_ (`alimony <https://github.com/alimony>`_)
+
+- Fix link markup `\#530 <https://github.com/django-compressor/django-compressor/pull/530>`_ (`merwok <https://github.com/merwok>`_)
+
+- Add support for COMPRESS\_CSS\_HASHING\_METHOD = None `\#526 <https://github.com/django-compressor/django-compressor/pull/526>`_ (`diox <https://github.com/diox>`_)
+
+- Remove compatibility with old 'staticfiles' app `\#525 <https://github.com/django-compressor/django-compressor/pull/525>`_ (`diox <https://github.com/diox>`_)
+
+- Use get\_template\(\) when dealing with django templates `\#524 <https://github.com/django-compressor/django-compressor/pull/524>`_ (`diox <https://github.com/diox>`_)
```

Comment:
Replace text with "In compress command, use get_template() instead of opening template files manually, fixing compatibility issues with custom template loaders".